.travis.yml

```diff
@@ -14,7 +14,7 @@ cache:
     - $HOME/.sbt
 
 script:
-  - travis_wait 45 scripts/travis-publish.sh
+  - travis_wait 60 scripts/travis-publish.sh
 
```

Comment:
Add the following ([see](https://github.com/InTheNow/catalysts/blob/master/.travis.yml#L3-L4))

```yml
install:
  - pip install --user codecov
```